os/kernel/binary_manager/compression/Make.defs

```diff
@@ -0,0 +1,24 @@
+###########################################################################
```

Comment:
I don't agree that this is in binary manager. Even if bin mgr uses it, it does not be a reason to be here. If it is, we should put loader into bin mgr as well.
And compression can be working without bin mgr. There is no reason to be working only with bin mgr.

Reply:
loader enabled, bin mgr enabled => compression can be working
loader enabled, bin mgr disabled => compression can be working
loader disabled, bin mgr disabled => compression is not working

Reply:
So, should I follow Kishore's advice and put code under TizenRT/os/compression.
Please advise regarding this.

And also regarding whether to put compression tool under tools/ or under os/tools/

Reply:
>> put code under TizenRT/os/compression.
I agree.

>> put compression tool under tools/ or under os/tools/
It's ok in os/tools. Because almost building tools are there.

Reply:
Ok..  will make these changes